flame/star/star_model.py

```
@@ -116,6 +122,40 @@ def _start_aggregator(self,
+            # Await intermediate results
+            result_dict = self.flame.await_intermediate_data(self.analyzer_ids)
+
+            print(f"\tReceived intermediate results from analyzers: {len(result_dict)}")
+
+            # Aggregate results
+            agg_res, converged, _ = self.aggregator.aggregate(list(result_dict.values()), simple_analysis)
+
+            print(f"\tAggregated result: {agg_res}, Converged: {converged}")
+
+            if converged:
+                response = self.flame.submit_final_result(agg_res, output_type, multiple_results)
```

Ruff (0.15.1)

[error] Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)